client/src/components/ShoeShowPage.js

```diff
@@ -1,7 +1,9 @@
 import React, { useState, useEffect } from "react"
+import ReviewList from "./ReviewList.js"
 
 const ShoeShowPage = props => {
     const [shoe, setShoe] = useState({})
+    const [reviews, setReviews] = useState([])
```

Comment:
alternatively, we should consolidate these 2 pieces of state. From the fetch request down below, you have:
```js
setShoe(shoeData.shoe) // this sets ALL of the `shoe` data in state for the show (including the nested `reviews`)
setReviews(shoeData.shoe.reviews) // this takes just the `reviews` array and sets it in state
```

^^ there is repetition in the data set in state.

Consider this approach:
```js
const [shoe, setShoe] = useState({
  reviews: []
})
```
^^ you do not need to define ALL properties for a shoe in the initial state, but we NEED to define the initial empty array for reviews so that they can be `map`ped over

```js
setShoe(shoeData.shoe) // initial state now has a place for the `reviews` to live, so we only need to set state once in the fetch request
```